app/views/shared/_header.html.erb

```diff
@@ -6,12 +6,12 @@
   </a>
       <div class="mediaflux">
         Mediaflux Status
-        <div class="mediaflux-status inactive"></div>
+        <div class="mediaflux-status unknown"></div>
```

Comment:
Display a brown circle when we have not checked the status (e.g. for a not logged in user)